driver-core/src/main/com/mongodb/internal/binding/SingleServerBinding.java

```diff
@@ -79,6 +73,11 @@ public SessionContext getSessionContext() {
         return NoOpSessionContext.INSTANCE;
     }
 
+    @Override
+    public ServerApi getServerApi() {
```

Comment:
Returns a nullable rather than an option.

Reply:
You want this to return `Optional<ServerApi>`?  

I think I didn't bother because it's not a public API

Reply:
Add @Nullable to all of these.

Done